### DISCUSSION TITLE
Can not login when instance is behind Cloudflare Access

### DISCUSSION THREAD

**Discussion by whollacsek** [Dokploy/dokploy#953] [Q&A]
### To Reproduce

1. Set up Cloudflare Access
1. Go to Dokploy login page
2. Login in works (green toast appears) but not redirection

Login works when disabling Cloudflare Access.

### Current vs. Expected behavior

Not sure which sub path is being blocked but we should not be stuck on login page

### Provide environment information

```bash
Dokploy: 0.15
```

### Which area(s) are affected? (Select all that apply)

Installation

### Are you deploying the applications where Dokploy is installed or on a remote server?

Same server where Dokploy is installed

### Will you send a PR to fix it?

No

**Comment by Siumauricio** [comment]
I don't understand, if the problem happens when you activate the cloudflare tunnels, I think the problem is there?

**Reply by whollacsek** [reply]
Hi, I'm using Cloudflare Access which doesn't need any setup server-side (unlike Cloudflare Tunnel). It works by granting users cookies and intercepting those cookies before forwarding to the target. It seems the web app keeps fail at polling this endpoint: `https://example.com/api/trpc/project.all,settings.isCloud,auth.get?batch=1&input=%7B%220%22%3A%7B%22json%22%3Anull%2C%22meta%22%3A%7B%22values%22%3A%5B%22undefined%22%5D%7D%7D%2C%221%22%3A%7B%22json%22%3Anull%2C%22meta%22%3A%7B%22values%22%3A%5B%22undefined%22%5D%7D%7D%2C%222%22%3A%7B%22json%22%3Anull%2C%22meta%22%3A%7B%22values%22%3A%5B%22undefined%22%5D%7D%7D%7D`

The decoded query param is:
`batch=1&input={"0":{"json":null,"meta":{"values":["undefined"]}},"1":{"json":null,"meta":{"values":["undefined"]}},"2":{"json":null,"meta":{"values":["undefined"]}}}`

If I disable Cloudflare Access and login 1 time then re-enable Access I can login (some times I need to submit the login form twice to be logged in) but the above endpoint keeps failing. This is my current workaround.

**Reply by whollacsek** [reply]
@Siumauricio why would this endpoint return undefined values?

**Reply by Ikram-Maulana** [reply]
@whollacsek I am also experiencing the same problem, is there a solution to this problem other than disabling cloudflare access then logging in to dokploy and then enabling cloudflare access again?

**Reply by whollacsek** [reply]
The solution is to change the polling behavior to make sure it carries the cookies but they don't seem to want to do it
